cross/build-android-rootfs.sh

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env bash
```

Comment:
This is an old version of the script. I'd suggest taking the latest version from corefx, which supports NDK 14 and ARM.